src/main/resources/api.yaml

```diff
@@ -207,3 +313,15 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  UserInProjectDto:
+    type: object
+    properties:
+      userId:
+        type: string
```

Comment:
w innych miejscach jest intem
(w sumie string jest bezpieczniejszy w js, ale może być int, byle spójny)

Reply:
gdzie jest intem?

Reply:
W sensie id jest intem, źle się wyraziłem

Reply:
Znaczy no tutaj jest stringiem, ja pytałem w jakich innych miejscach jest intem

Reply:
userId jest stringiem, projectId jest intem
Nie widzę sensu, żeby jedno było takie, a drugie takie. Może coś ustalmy, żeby wszędzie było spójne? (i tu, i w innych endpointach)

Reply:
Generalnie ja byłbym zawsze za stringiem, ale widziałem że było zaczęte intem więc używałem inta. Oprócz userów, bo jeśli przejdziemy w drugim sprincie na jakiś system logowania to jest duża szansa, że ID będzie stringiem.

Reply:
@kjarosh  Z tym nam się trochę bajzel robi, bo mamy już na przykład [to](https://github.com/papaya-agh/papaya-backend/blob/e677efad03ad5b6fa997049385a986f26544d169/papaya-core/src/main/java/pl/edu/agh/papaya/security/UserContext.java#L15), a Ty już chyba w kilku miejscach parsujesz stringa stworzonego z Longa na Longa (z wyjątkiem i tak dalej). Znaczy, z tego co rozumiem, idea jest szczytna, ale jak na razie w bazie jest Long i nie wiem, na ile reszta zespołu ogarnia Twoją wizję przyszłości

Reply:
Okay, to zmieniam na longa